### ISSUE TITLE
Missing options in Windows GUI

### ISSUE THREAD

**Issue by cfsmp3** [CCExtractor/ccextractor#979]
Missing stuff in CCExtractorGUI:

(hopefully some GSoC student will continue where I left it)

Output 3:
```
+   mprint ("   --segmentonkeyonly -key: When segmenting files, do it only after a I frame\n");
+	mprint ("                            trying to behave like FFmpeg\n\n");
mprint ("-sbs --splitbysentence: Split output text so each frame contains a complete");
+	mprint ("                       sentence. Timings are adjusted based on number of");
+	mprint ("                       characters.");
```

DVB (in output):
```
 	mprint ("            -dvbcolor: For DVB subtitles, also output the color of the\n");
 	mprint ("                       subtitles, if the output format is SRT or WebVTT.\n");
+	mprint ("          -nodvbcolor: In DVB subtitles, disable color in output.\n");
```

Input:
```
+	mprint ("-anvid --analyzevideo  Analyze the video stream even if it's not used for\n");
+	mprint ("                       subtitles. This allows to provide video information.\n");
+	mprint ("             -mkvlang: For MKV subtitles, select which language's caption\n");
+	mprint ("                       stream will be processed. e.g. 'eng' for English.\n");
+	mprint ("                       Language codes can be either the 3 letters bibliographic\n");
+	mprint ("                       ISO-639-2 form (like \"fre\" for french) or a language\n");
+	mprint ("                       code followed by a dash and a country code for specialities\n");
+	mprint ("                       in languages (like \"fre-ca\" for Canadian French).\n");
+	mprint ("  --webvtt-create-css: Create a separate file for CSS instead of inline.\n");
```

Don't know where to add it
```
    if (strcmp (argv[i],"-tickertext")==0 || strcmp (argv[i],"-tickertape")==0)

+		if (strcmp(argv[i], "-chapters") == 0){
+			opt->extract_chapters= 1;
+			continue;
+		}
```

DVB (in input):
```
+	mprint ("             -dvblang: For DVB subtitles, select which language's caption\n");
+	mprint ("                       stream will be processed. e.g. 'eng' for English.");
+	mprint ("                       If there are multiple languages, only this specified\n");
+	mprint ("                       language stream will be processed\n");
```

OCR:
```
+	mprint ("                 -oem: Select the OEM mode for Tesseract, could be 0, 1 or 2.\n");
+	mprint ("                       0: OEM_TESSERACT_ONLY - default value, the fastest mode.\n");
+	mprint ("                       1: OEM_LSTM_ONLY - use LSTM algorithm for recognition.\n");
+	mprint ("                       2: OEM_TESSERACT_LSTM_COMBINED - both algorithms.\n");
+	mprint ("             -ocrlang: Manually select the name of the Tesseract .traineddata\n");
+	mprint ("                       file. Helpful if you want to OCR a caption stream of\n");
+	mprint ("                       one language with the data of another language.\n");
+	mprint ("                       e.g. '-dvblang chs -ocrlang chi_tra' will decode the\n");
+	mprint ("                       Chinese (Simplified) caption stream but perform OCR\n");
+	mprint ("                       using the Chinese (Traditional) trained data");
+	mprint ("                       This option is also helpful when the traineddata file\n");
+	mprint ("                       has non standard names that don't follow ISO specs\n");
```

**Comment by dhrumilp15** [comment]
Were you thinking of making new tabs for these options or adding them as tiles of the output and input tabs?

**Comment by cfsmp3** [comment]
I'm open to everything including a complete rewrite of the GUI.

**Comment by dhrumilp15** [comment]
I'm also interested in something along those lines - I've been using WSL, but it would be nice to have a modern GUI. How did you get started with building the current one?

**Comment by cfsmp3** [comment]
> That would be useful - I've been using WSL, but it would be nice to have a modern GUI. How did you get started with building the current one?

I added them as they were being added to the command line program. And it shows.

**Comment by cfsmp3** [comment]
Closing, since there's a new GUI.
